lib/project_types/script/layers/application/create_script.rb

```diff
@@ -22,6 +23,7 @@ def call(ctx:, language:, script_name:, extension_point_type:, no_config_ui:)
                 .for(ctx, language, extension_point, script_name, project.id)
               install_dependencies(ctx, language, script_name, project_creator)
               bootstrap(ctx, project_creator)
+              script_project_repo.update_script_json(title: script_name, configuration_ui: !no_config_ui)
```

Comment:
The EPs will be defining the `script.json` file for each sample script, which wont be created until `bootstrap` is called. But there also needs to be a way to update the `script.json` after creation to include the `configurationUi` boolean which references the use of the `--no-config-ui` flag. 

Since we'll be using sparse-checkout to generate the samples, I don't see a way around this for now. However In the future, I think we could benefit from having some sort of templating solution (like using [mustache.js](https://github.com/janl/mustache.js/) for example) which could solve this problem because the `configurationUi` value could be a template that we replace.

Reply:
Ah, this all make sense.  Once we have sparse-checkout, `script.json` will be included in the **default** Script project.  Then, we can evolve to Mustache in the future if we need to.